docs/reference/repo_properties.rst

```diff
@@ -21,6 +21,31 @@ The following properties are supported for PPA-type repositories:
    - Examples:
       - ``ppa: snappy-devs/snapcraft-daily``
       - ``ppa: mozillateam/firefox-next``
+- priority
+   - Type: enum[string] or int
+   - Description: Overrides the default behavior when picking the source for a
+     particular package
+   - Format: ``always``, ``prefer`` or ``defer``. Alternatively an int other
+     than 0
+   - Notes: string equivalencies are ``always``: 1000; ``prefer``: 990;
+     ``defer``: 100
+   - Examples:
+      - ``priority: always``
+      - ``priority: 1000``
+- key-id
+   - Type: string
+   - Description: 40 character GPG key identifier ("long-form thumbprint" or
+     "fingerprint"). The Craft Application will first look for the corresponding
```

Comment:
I think we need to drop this sentence referring to `<project>/snap/keys` from these docs - it only applies to snapcraft